CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- `it` locale translations
```

Comment:
You should add a verb so it is clear what has been done with the `it` translations (fixed, added, etc...)